nelson-feng/wk02/1-mon/html-labs/busyhands/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+
+	<title>Busy Hands</title>
+	<!-- Font link -->
+	<!-- external CSS link -->
+	<link rel="stylesheet" href="css/normalize.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <nav>
+        <a class="navbar" href="Home">Home</a>
+        <a class="navbar" href="About">About</a>
+        <a class="navbar" href="Services">Services</a>
+        <a class="navbar" href="Contact Us">Contact Us</a>
+    </nav>
+
+    <div class = "top">
+        <!--Busy Hands Logo-->
+        <img src = "images/logo.png">
+        <!--Social Media tags-->
+        <div class id="socialMedia">
```

Comment:
To get these two elements sitting side-by-side, I'd start by setting both of their displays to `display: inline-block`. Alternatively, as I see you've used _flex_ elsewhere, you could play around with setting their **parent div** (the one with the class of `.top`) to `display: flex`;